containers/ecr-viewer/src/app/view-data/components/EcrDocument.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+import React, { useState } from "react";
+import { Accordion, Grid, GridContainer } from "@trussworks/react-uswds";
+import { AccordionItem } from "@/app/services/accordionItemService";
+import { ExpandCollapseButtons } from "./ExpandCollapseButtons";
+
+/**
+ * The body of the eCR document
+ * @param params React params
+ * @param params.initialAccordionItems Initial state of the accordion items
+ * @returns ecr document component
+ */
+export const EcrDocument = ({
+  initialAccordionItems,
+}: {
+  initialAccordionItems: AccordionItem[];
+}) => {
+  const [accordionItems, setAccordionItems] = useState(initialAccordionItems);
+  return (
+    <div className="margin-top-10">
+      <GridContainer className={"padding-0 margin-bottom-3 maxw-none"}>
+        <Grid row className="margin-bottom-05">
+          <Grid>
+            <h2 className="margin-bottom-0" id="ecr-document">
+              eCR Document
+            </h2>
+          </Grid>
+          <Grid className={"flex-align-self-center margin-left-auto"}>
+            <ExpandCollapseButtons
+              expandHandler={() =>
+                setAccordionItems(
+                  accordionItems.map((item) => ({ ...item, expanded: true })),
+                )
+              }
+              collapseHandler={() =>
+                setAccordionItems(
+                  accordionItems.map((item) => ({ ...item, expanded: false })),
+                )
+              }
+              descriptor="sections"
+            />
+          </Grid>
+        </Grid>
+        <div className="text-base-darker line-height-sans-5">
+          Displays entire eICR and RR documents to help you dig further into eCR
+          data
+        </div>
+      </GridContainer>
+      <Accordion
+        // HACK: get this to re-render when items change
```

Comment:
should we just make our own accordion component? or do we feel okay about this tradeoff?

Reply:
yes, we should (and did)